lib/figaro.rb

```diff
@@ -0,0 +1,62 @@
+require 'yaml'
+require 'erb'
+
+class Figaro
+  def self.env
+    Environment
```

Comment:
WDYT of making `Environment` no longer a singleton? Just an instance?

```suggestion
    @env ||= Environment.new
```

Or if we're worried about a race condition or somehting:


```ruby
  @env = Environment.new

  def self.env
    @env
  end
```

Reply:
Makes sense to me!